mpqp/core/instruction/gates/native_gates.py

```diff
@@ -479,6 +485,9 @@ def to_matrix(self) -> Matrix:
             ]
         )
 
+    def to_canonical_matrix(self):
```

Comment:
Return type missing ? (not mandatory, but we put it for to_matrix, so to stay coherent)

Reply:
done